uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/BookingPaymentView.kt

```diff
@@ -33,19 +33,23 @@ class BookingPaymentView @JvmOverloads constructor(context: Context,
 
     var paymentActions: BookingPaymentMVP.PaymentActions? = null
     var cardActions: BookingPaymentMVP.CardActions? = null
+    private var dropInActions: PaymentDropInMVP.View? = null
```

Comment:
would `dropInActions` be confusing as it is just a reference to the `View`

Reply:
I've rename to dropInView